libraries/eosiolib/dispatcher.hpp

```diff
@@ -114,10 +117,10 @@ namespace eosio {
  *
  * Example:
  * @code
- * EOSIO_ABI( eosio::bios, (setpriv)(setalimits)(setglimits)(setprods)(reqauth) )
+ * EOSIO_DISPATCH( eosio::bios, (setpriv)(setalimits)(setglimits)(setprods)(reqauth) )
  * @endcode
  */
-#define EOSIO_ABI( TYPE, MEMBERS ) \
+#define EOSIO_DISPATCH( TYPE, MEMBERS ) \
```

Comment:
Since dispatch logic is already changing (constructor/destructor now not called if there is no matching action) and this is a breaking minor release, I would like to clean this macro up further and get rid of the onerror stuff which really shouldn't belong here since it is a notification handler and not a regular action.

Contract developers already use custom dispatchers to handle notify handlers for things like `eosio.token::transfer`, so they could easily handle `eosio::onerror` handling (which is far less likely to be used).

Later when we have proper support for action _and_ notify handlers in CDT code generator, we will deprecate this EOSIO_DISPATCH macro anyway.